### ISSUE TITLE
Chart Usability

### ISSUE THREAD

**Issue by rmarcalo** [ScottLogic/StockFlux#666]
Hi,

After a few user testing sessions, I found that most problems were related with using the chart itself, rather than the new openfin features (which is understandable since there is a lot of complexity to it). I've made a few designs to explain and hopefully fix some of those usability issues, which are all shown in the following document: 

![chart usability](https://cloud.githubusercontent.com/assets/13066377/13821569/19bb00d6-eb9a-11e5-8630-5ef3cfdc1925.png)

![arrow](https://cloud.githubusercontent.com/assets/13066377/13821584/213c3aa0-eb9a-11e5-8001-5dbea5625804.png)
 

Please let me know if you have any questions regarding these designs, or if there any glaring mistakes I might have overlooked.

Rui

**Comment by owennw** [comment]
Thanks @rmarcalo! I think those suggestions will be very useful for us and BitFlux. Some of those features I think will be on the BitFlux team side, and some with us. 

@jleft I think you might want to take a look at this. I know at least the crosshair feature at the bottom is something that's been talked about already? But perhaps the x-axis scale and maybe even the icon might be of interest to push into BitFlux?

**Comment by ColinEberhardt** [comment]
Awesome feedback - thanks @rmarcalo

**Comment by jleft** [comment]
> Thanks @rmarcalo! I think those suggestions will be very useful for us and BitFlux. Some of those features I think will be on the BitFlux team side, and some with us.

Yes, thank you @rmarcalo - this is very valuable feedback for BitFlux :smiley: 

> @jleft I think you might want to take a look at this. I know at least the crosshair feature at the bottom is something that's been talked about already? But perhaps the x-axis scale and maybe even the icon might be of interest to push into BitFlux?

+ Yeah - we're planning on adding the crosshair to the indicator charts in a future release
+ I think changing the icon would be fine - I'll check with Courtney to makes sure she's happy with it from a BitFlux perspective
+ The axis tick labelling looks very cool  - I'd definitely like to try it!

**Comment by rmarcalo** [comment]
Thanks!

There was another issue with the summary chart at the bottom, where the right handle is cut by the borders of the chart, this made it harder for one of the users to know she could move the right-hand side of the scale as well as the first. 
![cut handle](https://cloud.githubusercontent.com/assets/13066377/13842096/4cc93ada-ec24-11e5-8351-e7063f214781.png)

It was also impossible for all users to zoom in to yesterday. It seems the chart is cutting directly to the value from 2 days before today. Is this a limitation of the data that lacks yesterday's values?
![no yesterday](https://cloud.githubusercontent.com/assets/13066377/13842186/cb3978ee-ec24-11e5-86e6-d4d50ce40edf.png)

**Comment by owennw** [comment]
@rmarcalo I can't access either of those images :/

**Comment by rmarcalo** [comment]
I updated them to PDFs, not sure why they weren't working (they appeared normally on my computer).

**Comment by jleft** [comment]
We have made some improvements in [BitFlux 1.6.0](https://github.com/ScottLogic/BitFlux/releases/tag/1.6.0), including:

- Back to present icon change
- Navigator brush handle is no longer cut

Other changes will be included in a later release.

**Comment by msuperina** [comment]
@jleft can this issue be closed here and moved to BitFlux ?
